submit-api/src/submit_api/services/invitation_service.py

```diff
@@ -46,7 +46,7 @@ def create_invitation(invite_data):
             session.flush()
             invitation = InvitationService._create_invitation_record(invite_data,
                                                                      role,
-                                                                     account.id,
+                                                                     account,
```

Comment:
dont u have to change the method to accept account instead of account id

Reply:
yes, because in that service I need to check if account has users to decide the is_first_time flag. And If I pass that separately to the method I get a too many arguments linting error